pkg/scheduler/framework/plugins/default_registry.go

```diff
@@ -53,8 +53,8 @@ type ConfigProducerRegistry struct {
 	PriorityToConfigProducer  map[string]ConfigProducer
 }
 
-// NewConfigProducerRegistry creates a new producer registry.
-func NewConfigProducerRegistry() *ConfigProducerRegistry {
+// NewDefaultConfigProducerRegistry creates a new producer registry.
+func NewDefaultConfigProducerRegistry() *ConfigProducerRegistry {
```

Comment:
It doesn't look like a Default so far

Reply:
We will add the registration of the various plugins in this function.

Reply:
Modified factory_test.go to start from an empty registry